plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveHadoop.java

```diff
@@ -93,11 +94,17 @@ public String runOnMetastore(String query)
         return executeInContainerFailOnError("mysql", "-D", "metastore", "-uroot", "-proot", "--batch", "--column-names=false", "-e", query).replaceAll("\n$", "");
     }
 
-    public HostAndPort getHiveMetastoreEndpoint()
+    public HostAndPort getHiveMetastoreHostAndPort()
```

Comment:
getHiveMetastoreHostAndPort is called from getHiveMetastoreEndpoint. Please inline.